src/hooks/firebase/index.ts

```diff
@@ -0,0 +1,68 @@
+import { initializeApp } from 'firebase/app';
+import {
+  getFirestore,
+  collection,
+  getDocs,
+  setDoc,
+  doc,
+} from 'firebase/firestore/lite';
+import { getStorage, ref, uploadString, getDownloadURL } from 'firebase/storage';
+import { DappItem, NewDappItem } from 'src/store/dapps-store/state';
+
+// firebase init - add your own config here
+const firebaseConfig = {
```

Comment:
the config should be stored as an environmental variable

Reply:
This code will be removed as soon as we switch to the admin service.